packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/RegisteredModels/RegisteredModelTableRow.scss

```diff
@@ -0,0 +1,20 @@
+/* Styling for section headers in kebab menu - target disabled dropdown items */
+.pf-v6-c-dropdown__menu-item.pf-m-disabled {
+  font-size: var(--pf-t--global--font--size--body--sm) !important;
+  color: var(--pf-t--global--text--color--subtle) !important;
+  font-weight: var(--pf-t--global--font--weight--body--bold) !important;
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+  padding-top: var(--pf-t--global--spacer--xs);
+  padding-bottom: var(--pf-t--global--spacer--xs);
+}
+
+/* Alternative: Target by content if needed */
+.pf-v6-c-dropdown__menu-item[aria-disabled="true"]:has-text("View model information"),
+.pf-v6-c-dropdown__menu-item[aria-disabled="true"]:has-text("Latest version actions") {
+  font-size: var(--pf-t--global--font--size--body--sm) !important;
+  color: var(--pf-t--global--text--color--subtle) !important;
+  font-weight: var(--pf-t--global--font--weight--body--bold) !important;
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+}
```

Comment:
Is it possible for us to use PF utility classNames to add these styles instead of CSS? I wonder if this will still look right in both Material and PF themes. We should try to avoid custom CSS if possible.